### DISCUSSION TITLE
Extract fragment intensities from MS3 scans

### DISCUSSION THREAD

**Discussion by phadi91hu** [mzmine/mzmine#1079] [Q&A]
Hello there!

 I'm totally new to MZmine and I'd like to do the following. I have a .raw file (Thermo) which contains MS1, MS2 and MS3 scans. I'd like to extract the intensities of two masses (m/z 69 and 86) from all MS3 scans belonging to different MS2 precursors. Is it possible? Is it possible to create an average intensity value for these extracted masses per MS2 precursors. I'll give you an example:
I have an ion with a value of m/z 500 in MS1 scan which is selected for MS2 fragmentation. m/z 86 is selected for MS3 from its MS2 fragments. I'd like to extract the intensities of two ions (m/z 69 and 86) from the MS3 scan of precursor m/z 500. This should be repeated for all MS2 precursors with MS3 scan. 

Thank you for your answer,
Adam

**Comment by robinschmid** [comment]
Extracting the intensity is currently not supported. We are still building more ways to interpret MSn data, but in general, it is very easy to contribute code to MZmine if you have ideas and someone with minimal coding experience in any language. 
Happy to help guide you/someone.
